packages/product-download-page/index.tsx

```diff
@@ -190,7 +189,7 @@ interface ProductDownloadsPageProps {
   packageManagerOverrides?: PackageManagerConfig[]
   showPackageManagers?: boolean
   pageTitle?: string
-  enterpriseMode: boolean
+  enterpriseMode?: boolean
```

Comment:
`enterpriseMode` already has a `false` default value, so can be safely set to optional without other changes.